README.md

```diff
@@ -80,10 +80,11 @@ The API is defined with Protocol Buffers/gRPC which can be found at https://gith
 
 See documentation in [Frontend Readme](./web/frontend/README.md).
 
-To start the client development environment, first run the same `go run` command mentioned in Building and Using, but with the environmental variable `ENV=development` (e.g. `ENV=development go run web/cmd/server/main.go -debug -config etc/configs/fake.json`). 
+To start the client development environment, first run the same `go run` command mentioned in Building and Using, but with the environmental variable `ENV=development` (e.g. `ENV=development go run web/cmd/server/main.go -debug -config etc/configs/fake.json`).
 
-Then navigate to `web/frontend` and run `npm start` in a new terminal tab. Visit `localhost:8080` to view the app, not `localhost:5173`. The latter is a hot module replacement server that rebuilds frontend asset changes.
+If you are in the `web/frontend` directory this is aliased as `npm run server`.
```

Comment:
I've added an `npm run server` alias for starting the developer server when doing client work. I was getting tired of looking up this command every time 😂